kubevirt_scripts/Containerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.fedoraproject.org/fedora:35
```

Comment:
That's EOL, is it intended?

Reply:
It's an example Dockerfile, but let me change it to something newer.